src/PetriEngine/Colored/Reduction/RedRulePreAgglomeration.cpp

```diff
@@ -0,0 +1,221 @@
+/*
+ * Authors:
+ *      Nicolaj Østerby Jensen
+ *      Jesper Adriaan van Diepen
+ *      Mathias Mehl Sørensen
+ */
+
+#include "PetriEngine/Colored/Reduction/RedRulePreAgglomeration.h"
+#include "PetriEngine/Colored/Reduction/ColoredReducer.h"
+
+namespace PetriEngine::Colored::Reduction {
+    bool RedRulePreAgglomeration::isApplicable(QueryType queryType, bool preserveLoops, bool preserveStutter) const {
+        return true;
```

Comment:
Agglomerationer virker kun for LTL\X
```suggestion
        return queryType != CTL && !preserveStutter;
```